src/main/java/com/ericsson/ei/rules/RulesHandler.java

```diff
@@ -103,6 +103,15 @@ public RulesObject getRulesForEvent(String event) {
         return null;
     }
 
+    /**
+     * Gets the active rules file content.
```

Comment:
Same here, with the reference to a file. Keep it as "Returns the active rules used by Eiffel Intelligence"

Reply:
Done.